src/curated-corpus/components/ScheduleDayData/ScheduleDayData.tsx

```diff
@@ -126,6 +132,36 @@ export const ScheduleDayData: React.FC<ScheduleDayDataProps> = (
     );
   };
 
+  /**
+   * Assigns the given grade to the given item.
+   *
+   * @param item ApprovedCorpusItem
+   * @param grade ApprovedItemGrade
+   * @param actionScreen ActionScreen
+   */
+  const assignGradeToItem = (
+    item: ApprovedCorpusItem,
+    grade: ApprovedItemGrade,
+    actionScreen: ActionScreen,
+  ): void => {
+    const variables = {
+      data: {
+        externalId: item.externalId,
+        grade,
+        actionScreen,
+      },
+    };
+
+    runMutation(
+      updateCorpusItemGrade,
+      { variables },
+      `Curated item "${item.title.substring(0, 40)}..." successfully graded`,
+      undefined,
+      undefined,
+      refetch,
+    );
+  };
```

Comment:
is this a good place to define this function?

the next PR for this will be to add the grade field to the item edit modal on all pages - which makes me think this function should be abstracted at a higher level, but i don't see an existing pattern for such a thing.

Reply:
I think if you look at the existing pattern & function above this, `moveItemToBottom`, you can leave it here

Reply:
I think it's fine. This component is where a big chunk of SchedulePage was moved out to. The plan is to split it further into testable components some time in the future 🤔